chromadb/test/property/test_client_url.py

```diff
@@ -0,0 +1,105 @@
+from typing import Optional
```

Comment:
I see you have become a hypothesis fan :)

Reply:
I was loving it. I was wondering where to put the test, so I left it in a separate file.